src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -73,13 +73,17 @@ public static String getSpecificMessageUsage(String field) {
             specificMessageUsage += "Finds students with stars that fall within the specified range.\n"
                     + "Parameters: OPERATOR NUMBER\n"
                     + "Example: " + COMMAND_WORD + " star < 1\n"
-                    + "Note: Valid operators are <, <=, >, >=, =. Only non-negative integers.";
+                    + "Note: Valid operators are <, <=, >, >=, =.\n"
+                    + "      Valid numbers are integers in range 0 to 50000.\n"
+                    + "      Invalid combinations are < 0 and > 50000 as they fall outside of the star range.";
```

Comment:
Would it be more accurate to say "as they do not overlap with the star range" instead?